app/js/main.js

```diff
@@ -658,7 +675,23 @@ function _getDefaultDirectory() {
 }
 
 /**
-* COnverting frames to png
+ * Create the default save directory if needed
+ */
+function createSaveDirectory() {
+    "use strict";
+    mkdirp(frameExportDirectory, function (err) {
+        if (err) {
+            console.error(`Failed to create directory at ${frameExportDirectory}`);
+            notifyError(`${frameExportDirectory} does not exist. Failed to create a directory at this location.`);
```

Comment:
Add `console.error(err);` before the call to `notifyInfo()` so we can know exactly what happened.